packages/zero-cache/src/scripts/run-query.ts

```diff
@@ -67,12 +80,21 @@ const host: QueryDelegate = {
       testLogConfig,
       '',
       db,
-      name,
-      schema.tables[name].columns,
-      schema.tables[name].primaryKey,
+      serverTableName,
+      Object.fromEntries(
```

Comment:
@darkgnotic added a helper for this common pattern:

https://github.com/rocicorp/mono/blob/4d9838dd48c78e73db2119a06865d41cf4c36d61/packages/shared/src/objects.ts#L25